packages/dds/map/src/mapKernel.ts

```diff
@@ -723,6 +744,8 @@ export class MapKernel implements IValueTypeCreator {
                 submit: (op: IMapValueTypeOperation, localOpMetadata: unknown) => {
                     this.submitMessage(op, localOpMetadata);
                 },
+                rebaseOp: (op: IMapDeleteOperation, localOpMetadata: unknown) => {
+                },
```

Comment:
I think (if I got comments above right), we should never get here, as this is used for local values that we removed from code. If so, having assert here would be nice.